packages/devextreme/js/__internal/core/ai_integration/commands/generateColumn.ts

```diff
@@ -0,0 +1,33 @@
+import type { GenerateColumnCommandParams, GenerateColumnCommandResult } from '@js/common/ai-integration';
+import { BaseCommand } from '@ts/core/ai_integration/commands/base';
+import type { PromptData, PromptTemplateName } from '@ts/core/ai_integration/core/prompt_manager';
+
+export class GenerateColumnCommand extends BaseCommand<
+  GenerateColumnCommandParams,
+  GenerateColumnCommandResult
+> {
+  protected getTemplateName(): PromptTemplateName {
+    return 'generateColumn';
+  }
+
+  protected buildPromptData(params: GenerateColumnCommandParams): PromptData {
+    const dataDescription = this.generateDataDescription(params.data);
+    return {
+      user: {
+        text: params.text,
+        data: dataDescription,
+      },
+    };
+  }
+
+  protected parseResult(response: string): GenerateColumnCommandResult {
+    let result: GenerateColumnCommandResult = {};
```

Comment:
[suggestion: minor]
Why let?

```suggestion
    const result: GenerateColumnCommandResult = JSON.parse(response);
    return result;
```